### DISCUSSION TITLE
Choose an option : update this label

### DISCUSSION THREAD

**Discussion by NicolasFabre81** [Budibase/budibase#7799] [Help] [answered]
Is there a way to update and/or translate this label ?
<img width="372" alt="image" src="https://user-images.githubusercontent.com/28592069/190594162-27193b91-2430-4cbb-b70b-39053ccae167.png">

**Comment by melohagan** [comment, marked as answer]
Is that an options picker? There's a setting to change the placeholder:
![Screenshot 2022-09-16 at 11 28 57](https://user-images.githubusercontent.com/101575380/190619261-87d2a116-6e9f-47f6-b92c-3e4c5cf1986f.png)

**Reply by NicolasFabre81** [reply]
Damned !
